admin/led

```diff
@@ -16,10 +18,10 @@ DBUS_PROPERTY="Asserted"
 # parse command line options
 ACTION=
 while [[ $# -gt 0 ]]; do
-  case $1 in
+  case "$1" in
     -h | --help)
       echo "Chassis LED identify control."
-      echo "Use: ${0##*/} [OPTION...] ACTION"
+      echo "Use: $0 [OPTION...] ACTION"
```

Comment:
Ehm... Why did you remove the basename substitution?

Reply:
We don't need that, I suppose we even shouldn't remove the path. You can have multiple programs with the same name in linux. This basename came from my experiments with chroot, where I have to hide the real executable path and work over some kind of proxy.